libs/application/api/files/src/lib/file.service.ts

```
@@ -257,10 +257,10 @@ export class FileService {
 
         try {
           this.logger.info(`Deleting attachment ${s3key} from bucket ${bucket}`)
-          await this.awsService.deleteObject({ bucket, key: s3key })
+          await this.s3Service.deleteObject({ bucket, key: s3key })
           result = {
             ...result,
-            deleted: result.deleted++,
+            deleted: result.deleted += 1,
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)